packages/arb-token-bridge-ui/src/components/TransferPanel/AdvancedSettings.tsx

```diff
@@ -1,11 +1,35 @@
 import { useEffect, useState } from 'react'
+import { isAddress } from 'ethers/lib/utils'
 import { ChevronDownIcon, ChevronUpIcon } from '@heroicons/react/24/outline'
```

Comment:
NIT: We should probably keep the chevron icon invisible in the condition where AdvancedSetting collapse is not optional. We can define a variable for the logic in line 95 -` if (isSmartContractWallet || destinationAddress)` and reuse it.